test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -223,6 +214,71 @@ TEST(help_add_test, add_option)
                            "SeqAn version: 3.0.0");
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
```

Comment:
use view::filter(!...) instead view::remove_if(...) because the latter is deprecated.